packages/nimble-components/src/table/styles.ts

```diff
@@ -21,6 +21,10 @@ export const styles = css`
         height: 480px;
     }
 
+    .disable-select {
+        user-select: none;
```

Comment:
@jattasNI pointed out that this doesn't work correctly on Safari. We need to also set `-webkit-user-select: none;`.

```suggestion
        user-select: none;
        -webkit-user-select: none;
```

Reply:
If you update from main, you can now use the shared `userSelectNone` styling.

Reply:
Done